src/main/java/amazon/review/model/ReviewDto.java

```diff
@@ -0,0 +1,18 @@
+package amazon.review.model;
+
+import java.time.LocalDateTime;
+import lombok.Data;
+
+@Data
+public class ReviewDto {
```

Comment:
```suggestion
public class ReviewCsvDto {
```
usually, you have different dtos and it's good to be specific

Reply:
done